### ISSUE TITLE
Training on my own dataset

### ISSUE THREAD

**Issue by yangyvhang666** [ZiweiWangTHU/BiDet#2]
It's nice to see your work,If i want to train on my own dataset,which parts should i modify

**Comment by Wuziyi616** [comment]
Thank you for your attention on our work! I think there are two parts you should modify to train on your own dataset (I will take SSD300 for example, Faster R-CNN can be modified in similar way):

1. ssd/data/config, this file contains training configuration. So you should add a specific configuration for your own dataset, for example 'num_classes', 'lr_steps' for learning rate decay, etc.

2. Still in ssd/data, I think you should write a Dataset class inheriting torch.utils.data.Dataset to load your own dataset. I think you can use the same augmentation as VOC or COCO so you only need to read in image and label, but transform label might a little troublesome :)

After doing so, I think you can write a training file and use our BiDet-SSD to train on your own dataset! Wish you good luck and hope our work can help you~
